app/routers/event.py

```diff
@@ -174,6 +174,11 @@ def get_event_with_editable_fields_only(event: Dict[str, Any]
 def _update_event(db: Session, event_id: int, event_to_update: Dict) -> Event:
     try:
         # Update database
+        print(event_to_update)
+        location = get_location_coordinates(event_to_update.get('location'))
```

Comment:
Not sure if there is a location. It is best to wrap in if.
I think its location is more appropriate for the second function - update_event, because the function here only does the update operation.